core/src/test/scala/unit/kafka/utils/TestUtils.scala

```diff
@@ -455,7 +455,7 @@ object TestUtils extends Logging {
    */
   def createNewProducer(brokerList: String,
                         acks: Int = -1,
-                        metadataFetchTimeout: Long = 3000L,
+                        maxBlockMs: Long = Long.MaxValue,
                         blockOnBufferFull: Boolean = true,
```

Comment:
Seems this variable is not used any more, could we remove it from createNewProducer as well?